html/controls/lt/lt-sheet.js

```diff
@@ -15,10 +19,30 @@ function prepareLtSheetTable(element, teamId, mode) {
 				element.find('#head').css('color', WS.state['ScoreBoard.Team(' + teamId + ').Color(operator_fg)']);
 			});
 		}
-		
-		WS.Register(['ScoreBoard.CurrentPeriodNumber'], function(k, v) { createPeriod(v); });
+
+		WS.Register(['ScoreBoard.Jam']); // for fielding editor
+		WS.Register(['ScoreBoard.Period'], handleUpdate);
+
+		if (mode == 'plt') {
+			WS.Register(['ScoreBoard.UpcomingJamNumber'], function (k, v) { element.find("#upcoming .JamNumber").text(v); });
```

Comment:
You did introduce quite a few DOM lookups in this part. And most of them for elements that change frequently.

Reply:
They're not per-skater or per-penalty, so they don't affect load times notably and everything else I profiled was fast enough not to worry.